Documentation/getting-started.md

```diff
@@ -28,59 +28,39 @@ If there is a missing package, debuild will tell you which package is missing.
 If you haven't configured debuild properly, yet, you will need to add ```-uc -us``` options to ```debuild```.
 
 
-**Approach 2.** Build with Cmake
-
-At the git repo root directory,
-```bash
-$ mkdir -p build  # We recommend to build in a "build" directory
-$ cd build
-$ rm -rf *        # Ensure the build directory is empty
-$ cmake ..
-$ make
-$ cd ..
-```
-
-You may copy the resulting plugin (.so file) to gstreamer plugin repository. Or do
-```bash
-$ cd build
-$ sudo make install
-```
-if installing NNstreamer plugin libraries into ```%{_libdir}```.
-
-
-**Approach 3.** Build with meson
+**Approach 2.** Build with meson
 * https://mesonbuild.com/Getting-meson.html
 
-install the required packages.
+Install the required packages. (meson >= 0.40)
 
-```bash
+```
```

Comment:
@jaeyun-jung  Why do you  want to replace "```bash" with "```"?  In my case, I much prefer the existing style to the change.

Reply:
I thought the optional language identifier is unnecessary.

Reply:
Let's keep the existing style because the statements is shell command.

Reply:
Now re-updated with identifier.